src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import { BirthdayCountdown } from "./components/Countdown";
+import { DvdRichard } from "./components/DvdRichard";
 
 function App() {
 
   return (
     <div>
       <BirthdayCountdown />
+      <div style={{
```

Comment:
Why did you decide to add a style inline over importing in App.css?

Reply:
Mostly just preference, they are functionally the same.

Reply:
I can move it to the css file